keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md

```diff
@@ -756,12 +996,11 @@ question.
 Pick one more of these and delete the rest.
 -->
 
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+N/A - no SLI defined
```

Comment:
Sorry for confusion - I meant adding this for the question above (for SLO) - if we don't have SLIs, then SLO doesn't make sense.